### DISCUSSION TITLE
Seamonkey icon request

### DISCUSSION THREAD

**Discussion by NicosArv** [elrumo/macOS_Big_Sur_icons_replacements#1480] [Icon request]
https://www.seamonkey-project.org

I've searched and most are unapologeticlly round, like the original and do not fit in with modern macOS icons. I found one squared off icon [here](https://www.iconspedia.com/icon/seamonkey-icon-40456.html), but it's low quality, not exactly the correct style, and is limited to 256x256.  I'd love to see a higher quality icon for use here if someone has the time? 

-Many thanks, Nick

**Comment by vikasbhatia** [comment]
I'll give this a go in the next day or so. Please @ me and give me a nudge if I haven't responded before then.

**Comment by vikasbhatia** [comment]
Okay, here you go! I've just uploaded it so the ICNS will be available [on my profile](https://macosicons.com/#/u/sidewinder) as soon as it's approved. Adding the PNG below. I think I've recreated the look and feel pretty faithfully off the SVG logo I found on Wikipedia, but I'm happy to make any edits to it if you want me to.

<img width="256" alt="SeaMonkey" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/ea1a7926-d65f-43cb-b20d-fe95ebd0246b">

**Reply by NicosArv** [reply]
Thank you, it's very much appreciated!

-Nick
